File: query-dsl-java/src/main/java/com/example/querydsljava/dao/repository/PhoneRepository.java
package com.example.querydsljava.dao.repository;

import com.example.querydsljava.dao.entity.PhoneEntity;
import org.springframework.data.jpa.repository.JpaRepository;
import org.springframework.data.querydsl.QuerydslPredicateExecutor;

public interface PhoneRepository extends JpaRepository<PhoneEntity, String>, QuerydslPredicateExecutor<PhoneEntity> {

}

File: query-dsl-kotlin/src/main/kotlin/com/example/querydslkotlin/dao/repository/PhoneRepository.kt
package com.example.querydslkotlin.dao.repository

import com.example.querydslkotlin.dao.entity.PhoneEntity
import org.springframework.data.jpa.repository.JpaRepository
import org.springframework.data.querydsl.QuerydslPredicateExecutor

interface PhoneRepository: JpaRepository<PhoneEntity, String>, QuerydslPredicateExecutor<PhoneEntity>